app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -90,27 +90,29 @@ private fun HomeSectionHeaderContent(
 ) {
     Row(
         modifier = Modifier.fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically,
     ) {
         SectionHeader(
             text = headerText,
             textColor = textColor,
             modifier = Modifier
                 .weight(1f)
-                .wrapContentHeight(align = Alignment.Top)
+                .wrapContentHeight(),
         )
 
         onShowAllClick?.let {
-            ClickableText(
-                text = AnnotatedString(text = stringResource(id = R.string.recent_tabs_show_all)),
-                modifier = Modifier.padding(start = 16.dp)
+            BasicText(
```

Comment:
I am wondering why we are replacing `ClickableText` since it is just a wrapper around `BasicText`

Reply:
I believe this is because the `onClick` implementation for `ClickableText` is made using `Modifier.pointerInput`, whereas `Modifier.clickable` allows talkback to view the modified composable as a button.

I noticed this is also used in [`ClickableSubstringLink`](https://github.com/mozilla-mobile/fenix/blob/aa4a072d73a036b7108740d53a466daadafed397/app/src/main/java/org/mozilla/fenix/compose/ClickableSubstringLink.kt#L81), should we file an issue for that? the links provided by this component are not accessible either.

Reply:
The `ClickableSubstringLink` used for Pocket is merged with the other siblings so there is just one a11y item which does allow double click to open the link.
The same could be done for the `ClickableSubstringLink` used for wallpaper settings but I'm not seeing a link there.